server/pkg/controller/mailing_lists.go

```diff
@@ -76,45 +92,68 @@ func NewMailingListsController() *MailingListsController {
 // the email addresses of our customers in a Zoho Campaign "list", and subscribe
 // or unsubscribe them to this list.
 func (c *MailingListsController) Subscribe(email string) error {
-	if c.shouldSkip() {
-		return stacktrace.Propagate(ente.ErrNotImplemented, "")
+	if !(c.shouldSkipZoho()) {
+		// Need to set "Signup Form Disabled" in the list settings since we use this
+		// list to keep track of emails that have already been verified.
+		//
+		// > You can use this API to add contacts to your mailing lists. For signup
+		//   form enabled mailing lists, the contacts will receive a confirmation
+		//   email. For signup form disabled lists, contacts will be added without
+		//   any confirmations.
+		//
+		// https://www.zoho.com/campaigns/help/developers/contact-subscribe.html
+		return c.doListActionZoho("listsubscribe", email)
```

Comment:
So the issue we discussed offline will get triggered here as well, since you're `return`-ing. If both Zoho and Listmonk are configured, we will return after performing the action for Zoho.

Instead, we should do Zoho, return an error if there's any. Then do Listmonk, return an error if there's any. And finally return `nil`.